drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua

```diff
@@ -0,0 +1,44 @@
+local zcl_commands = require "st.zigbee.zcl.global_commands"
+local aqara_utils = require "aqara/aqara_utils"
+
+local FINGERPRINTS = {
+  { mfr = "LUMI", model = "lumi.motion.agl02" },
+  { mfr = "LUMI", model = "lumi.motion.agl04" }
+}
+
+local is_aqara_products = function(opts, driver, device)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local aqara_motion_handler = {
```

Comment:
it seems same name `aqara_motion_handler` for all sub drivers also.  @sbkimaqara  Could you check this is working as expected?

Reply:
The device worked correctly. But I will modify the code according to your advice.

Reply:
thanks. but if it works. I'm just fine.